client/cypress/cypress.js

```diff
@@ -14,31 +12,49 @@ try {
 
```

Comment:
@eradman Would you have time to look over this cypress.js file?

The changes in it are more complex than I'm able to really parse atm.

The changes in the Cypress tests below though seem workable.  And it'd be good to get the Cypress pieces updated past Cypress 12.x. :smile: